### ISSUE TITLE
Component: Micro Footer - On Select by location or language, search edit field, the results are getting updated dynamically, but screen reader user would not be aware

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#3818]
## Detailed description
On Firefox JAWS & iPhone VO

Issue : As we provide the input in the search edit field, the results are getting updated dynamically. But, Screen reader user would not be aware of this.

Expected:
1. Screen reader should announce the number of results as they dynamically update. 
2. Provide the following instruction and Programmatically Associate using aria-described by
"Note: Location/Language will be dynamically updated as you provide the input".
